docker/Dockerfile

```diff
@@ -41,38 +41,38 @@ RUN --mount=type=cache,target=/lemmy/target set -ex; \
     fi
 
 # ARM64 builder
```

Comment:
Commenting out the broken arm builds until we get an updated rust arm image from @raskyld

Reply:
Hey! Thanks for pinging me, I launched a build with v0.2.0 tag :)

Reply:
Thx! I'll get another tag started to test it shortly.